### PR TITLE
HTTP_Engine: Mark GetRequest as obsolete and change back to public

### LINKED ISSUE #34: Changing GetRequest(string uri) to a private method is breaking things
#### Description:
Change to a private method on ``GetRequest(string uri)`` breaks what was a handy tool. Can we switch back to public and deprecate it. 
I realise this relates to use of Push / Pull adaptor methods and a public ``GetRequest`` breaks the rules

### PR DESCRIPTION
**Pull request by rolyhudson**
Closes #34 
marks the GetRequest method as obsolete with explanation and allows continued use by changing the method to public

### CODE REVIEW

File: HTTP_Engine/Compute/GetRequest.cs [hunk 1]
```diff
@@ -31,8 +31,8 @@ public static partial class Compute
         /***************************************************/
         /**** Private  Methods                          ****/
         /***************************************************/
-
-        private static string GetRequest(string uri)
+        [System.Obsolete("GetRequest is deprecated, pit does not comply with BHoM Adaptor Push and Pull protocols")]
```
Comment (epignatelli) [inline HTTP_Engine/Compute/GetRequest.cs]: Can you use `BH.oM.Reflection.Attributes.Deprecated` instead of `System.Obsolete?

Comment (rolyhudson) [reply HTTP_Engine/Compute/GetRequest.cs]: Yep no problem
``[Deprecated("3.1", "Does not comply with BHoM Adaptor Push and Pull protocols")]``
fromVerision = 3.1?

**Comment by rolyhudson** [comment]
@FraserGreenroyd labels added 👍 only thing I'm not %100 sure on is the meaning of ``fromVersion`` in the sense of deprecation. Guessing its from 3.1

**Comment by FraserGreenroyd** [comment]
> @FraserGreenroyd labels added 👍 only thing I'm not %100 sure on is the meaning of `fromVersion` in the sense of deprecation. Guessing its from 3.1

Yeah that's correct - deprecated from version is the milestone we're working on so that by the time we get to beta release, that's when it's fully 'deprecated' for those only working to beta versions :smile:

File: HTTP_Engine/Compute/GetRequest.cs (hunk 1 above)
Comment (epignatelli) [reply HTTP_Engine/Compute/GetRequest.cs]: Yep! Correct!

**Pull request merged**